main.js

```diff
@@ -138,11 +138,24 @@ define(function (require, exports, module) {
 
     // Function to run when the menu item is clicked
     function promptForApiKey() {
-        var apikey = window.prompt("[WakaTime] Enter your wakatime.com api key:", prefs.get("apikey"));
-        if (apikey) {
-            prefs.set("apikey", apikey);
-            prefs.save();
-        }
+
+        var dialog, wakaKey, $input, $okButton;
+
+        dialog  = Dialogs.showModalDialog('DefaultDialogs', 'Enter Wakatime key',
```

Comment:
We should mention the api key comes from wakatime.com or wakatime.com/settings here.